backend/lib/edgehog/triggers/handler/manual_actions/handle_trigger.ex

```diff
@@ -41,6 +41,14 @@ defmodule Edgehog.Triggers.Handler.ManualActions.HandleTrigger do
   @ota_response "io.edgehog.devicemanager.OTAResponse"
   @system_info "io.edgehog.devicemanager.SystemInfo"
 
+  @initial_states [
```

Comment:
probably not needed but to be extra safe:
```suggestion
  @initial_states [
    :created,
```